src/bin/vip-import-validate-files.js

```diff
@@ -64,95 +69,65 @@ command( { requiredArgs: 1, format: true } )
 		}
 
 		/**
-		 * Media file extension validation
-		 *
-		 * Ensure that prohibited media file types are not used
+		 * Get Media Import configuration
 		 */
+		const mediaImportConfig = await getMediaImportConfig();
 
-		// Collect invalid files for error logging
-		let intermediateImagesTotal = 0;
-
-		const errorFileTypes = [];
-		const errorFileNames = [];
-		const intermediateImages = {};
-
-		// Iterate through each file to isolate the extension name
-		for ( const file of files ) {
-			// Check if file is a directory
-			// eslint-disable-next-line no-await-in-loop
-			const stats = await fs.promises.stat( file );
-			const isFolder = stats.isDirectory();
-
-			const extension = path.extname( file ); // Extract the extension of the file
-			const ext = extension.substr( 1 ); // We only want the ext name minus the period (e.g- .jpg -> jpg)
-			const extLowerCase = ext.toLowerCase(); // Change any uppercase extensions to lowercase
-
-			// Check for any invalid file extensions
-			// Returns true if ext is valid; false if invalid
-			const validExtensions = acceptedExtensions.includes( extLowerCase );
-
-			// Collect files that have no extension, have invalid extensions,
-			// or are directories for error logging
-			if ( ! extension || ! validExtensions || isFolder ) {
-				errorFileTypes.push( file );
-			}
-
-			/**
-			 * Filename validation
-			 *
-			 * Ensure that filenames don't contain prohibited characters
-			 */
-
-			// Collect files that have invalid file names for error logging
-			if ( isFileSanitized( file ) ) {
-				errorFileNames.push( file );
-			}
-
-			/**
-			 * Intermediate image validation
-			 *
-			 * Detect any intermediate images.
-			 *
-			 * Intermediate images are copies of images that are resized, so you may have multiples of the same image.
-			 * You can resize an image directly on VIP so intermediate images are not necessary.
-			 */
-			const original = doesImageHaveExistingSource( file );
-
-			// If an image is an intermediate image, increment the total number and
-			// populate key/value pairs of the original image and intermediate image(s)
-			if ( original ) {
-				intermediateImagesTotal++;
-
-				if ( intermediateImages[ original ] ) {
-					// Key: original image, value: intermediate image(s)
-					intermediateImages[ original ] = `${ intermediateImages[ original ] }, ${ file }`;
-				} else {
-					intermediateImages[ original ] = file;
-				}
-			}
+		if ( ! mediaImportConfig ) {
+			console.error( chalk.red( '✕ Error:' ), 'Media import configuration not available' );
```

Comment:
```suggestion
			console.error( chalk.red( '✕ Error:' ), 'Could not retrieve validation metadata. Please contact VIP Support.' );
```

While the term "media import config" is informative to us, not sure if it is for the customer. We can keep it, and rephrase it a bit.

Reply:
Thanks, Sarosh! Addressed in c060b629dbac15c790970a139ccd77855519f9fd